src/android/gradle.ts

```diff
@@ -45,12 +45,20 @@ export async function selectAppFile(
 
   if (appFiles.length === 0) {
     Sentry.setTag('custom-build-logic', true);
-    const appFile = await abortIfCancelled(
+    let appFile = await abortIfCancelled(
       clack.text({
         message: `Unable to find your app's directory. 
         Please enter the relative path to your app's build.gradle file from the root project (e.g. "app/build.gradle.kts")`,
       }),
     );
+    while (!appFile.includes('.gradle') || !fs.existsSync(appFile)) {
```

Comment:
l: can we solve this with clack's [`validate` option](https://github.com/natemoo-re/clack/tree/main/packages/prompts#text)?

Reply:
nice, didn't know that existed